### ISSUE TITLE
Server Status in Server Übersicht anzeigen

### ISSUE THREAD

**Issue by vogelor** [mittwald/feature-requests#56]
**Welches Problem möchtest du lösen? Wann tritt es auf?**
Ich kann nicht auf einen Blick sehen, welchen "Status" mein(e) Server hat

**Welche Lösungsideen hast du?**
Man könnte in der Server-Liste (so wie z.B. bei der Domain Liste) einen farbigen Punkt anzeigen:
Grün = Alles O.K.
Gelb = Auslastung über 80% oder Platte zu 80% voll (oder andere % Zahl)
Rot = Auslastung über 95% (oder so ähnlich)

Dann müsste ich nur immer wieder in die Übersicht und wüsste sofort, was los ist bzw. welchen Server ich mir ansehen musss.
D.h. ich spare mir den BLick ins Dashboard

Eine Benachrichtigung (z.B. per E-Mail wäre auch super)

**Hast du zusätzliche Informationen (wie z.B. Screenshots)?**
Nein

**Comment by patrickhilker** [comment]
Konkret geht es dir um eine Mini-Status-Anzeige in der Server-/Projekt-Liste, das verstehe ich richtig, oder?

**Comment by vogelor** [comment]
Genau, damit ich einfach in der Übersicht schon den Status sehe, OHNE (wenn ich mehrere Server habe) auf jeden einzelnen Server klicken muss.
NOCH BESSER wäre natürlich, wenn das gleich im Dashboard stehen würden. Also eine eigene Kachel "Deine Server-Stati"

**Comment by vogelor** [comment]
Wenn ich dort sehe, "irgendwas" stimmt nicht, dann kann ich ja in den Server gehen und mir die Details ansehen.
Das wäre übrigens auch toll bei den Projekten. Ich klicke in die Projekte und sehe schon in der Liste, welche mir Probleme machen könnten. Nur weiß ich bei den Projekten echt nicht, was genau "Probleme" sind. Beim Server ist das einfacher, da diese ja definierte Ressourcen haben...
